src/contrib/cluster/Akka.Cluster.Tools/Client/ClusterClientExtensions.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Akka.Actor;
+
+namespace Akka.Cluster.Tools.Client
+{
+    public static class ClusterClientExtensions
+    {
+        public static Action Subscribe<T>(this IActorRef client, ClusterClient.Subscribe subscribe)
```

Comment:
I understand reason behind this, but without having typed `IActorRef` this is risky API (this method can be called on any `IActorRef` not only on cluster client). We usually leave this kind of extension methods to be implemented by users for their convinience and keep message passing part explicit - not hidding it behind methods.